index.html

```diff
@@ -0,0 +1,30 @@
+<html>
+
+<head>
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+
+</head>
+
+<body>
+    <div class="menu-bar">
+
+        <ul>
+            <li class="active"><a href="index.html"><i class="fa fa-home"></i>Henry Giathi</a></li>
+            <div class="nav-wrapper">
+                <li><a href="About.html"><i class="fa fa-user"></i>About</a>
+            </div>
+
+            <li><a href="https://github.com/Wawesh2020"><i class="fa fa-link"></i>Github</a>
+                <div class="sub-menu-1">
+            </li>
+            <li><a href="https://www.linkedin.com/in/henry-giathi-7625441a3/"><i class="fa fa-linkedin"></i>Linkedin</a>
+            </li>
```

Comment:
The web standard is for pages within your site stay in the same tab but links off to another site open a new tab